libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -302,6 +306,26 @@ export type DependentEntitiesGraphResponse = DependentEntitiesResponse;
  */
 export type WorkspaceDataFiltersLayout = DeclarativeWorkspaceDataFilters;
 
+/**
+ * @internal
+ */
+export type WorkspaceDataFiltersResult = JsonApiWorkspaceDataFilterOutDocument;
```

Comment:
Please rename `WorkspaceDataFiltersResult` → `WorkspaceDataFilterResult` (without any `s` after `Filter`).

`WorkspaceDataFiltersResult` can make people think that the type is some sort of array, can't it?